### ISSUE TITLE
PHP session handler problem

### ISSUE THREAD

**Issue by anniballo** [aws/aws-sdk-php#1185]
Hello,
could someone help me with this problem?
98% or our session traffic works fine with DynamoDB but sometimes I get this problem with the serialization.
The referrer is always the same and the problem happens just for a few of them.
Any clues?
Thank you so much! :)

PHP Warning:  Error writing session rasid_im_id_obscured: Error executing \"UpdateItem\" on \"https://dynamodb.eu-west-1.amazonaws.com\"; AWS HTTP error: Client error: `POST https://dynamodb.eu-west-1.amazonaws.com` resulted in a `400 Bad Request` response:\\n{\"__type\":\"com.amazon.coral.service#SerializationException\"}\\n SerializationException (client):  - {\"__type\":\"com.amazon.coral.service#SerializationException\"} in /riseart/app/vendor/aws/aws-sdk-php/src/DynamoDb/StandardSessionConnection.php on line 145, referer: http://statgen.ncsu.edu"

**Comment by cjyclaire** [comment]
@anniballo I noticed you have mentioned "the problem happens just for a few of them", are you suggesting the error occurs with several specific request only? Could you provide some code snippets that might help us reproduce the issue? Also, may I ask the PHP version that you are using?

Meanwhile, our [`README.md#Getting Help`](https://github.com/aws/aws-sdk-php#getting-help) has been updated recently, feel free to get engaged in other channels to get help as well.

**Comment by anniballo** [comment]
Hello,
thank you so much for your interest :)
I did a bit of research and tests on my log files during the weekend and I've discovered that happens just with a specific request.
That request comes from a malicious bot who looking for vulnerabilities on the server, injects weird code in the user agent or inside the HTTP header fields.

We have some information about the user agent in the session and that drives crazy our application.

So, basically, it's not the framework's issue :)

Again
Thank you for your time and sorry for the useless post.
